prow/repoowners/repoowners_test.go

```diff
@@ -1264,11 +1264,10 @@ reviewers:
 		if test.expected != s {
 			t.Errorf("result '%s' is differ from expected: '%s'", s, test.expected)
 		}
-		simple, err := LoadSimpleConfig(b)
-		if err != nil {
+
+		if simple, err := LoadSimpleConfig(b); err != nil {
 			t.Errorf("unexpected error when load simple config: %v", err)
-		}
-		if !reflect.DeepEqual(simple, test.given) {
+		} else if !reflect.DeepEqual(simple, test.given) {
```

Comment:
What is this change fixing? AFAICT both `simple` and `err` were already being assigned properly.

Reply:
I don't really remember why I made this change, but as you said, it's not needed.